src/01_bignum.py

```diff
@@ -1,4 +1,5 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
```

Comment:
Could also be done using the `**` operator, ex `num1 ** power_num`. Great job finding the `pow` method!